recipes-devtools/valgrind/patches/0002-xen-v4v-Add-the-v4v-chardev-ioctls.patch

```diff
@@ -0,0 +1,250 @@
+From 437462a40e97c2e0a2aca9e5f64058681ca4385b Mon Sep 17 00:00:00 2001
+From: Eric Chanudet <chanudete@ainfosec.com>
+Date: Fri, 15 Feb 2019 21:36:46 -0500
+Subject: [PATCH 2/2] xen/v4v: Add the v4v chardev ioctls.
+
+Check arguments of ioctls used by the V4V chardev as released in OpenXT
+8.
+
+Signed-off-by: Eric Chanudet <chanudete@ainfosec.com>
+---
+ coregrind/m_syswrap/syswrap-linux.c | 144 +++++++++++++++++++++++++++-
+ include/vki/vki-linux.h             |  53 ++++++++++
+ 2 files changed, 196 insertions(+), 1 deletion(-)
+
+diff --git a/coregrind/m_syswrap/syswrap-linux.c b/coregrind/m_syswrap/syswrap-linux.c
+index f2ebc161f..b51055162 100644
+--- a/coregrind/m_syswrap/syswrap-linux.c
++++ b/coregrind/m_syswrap/syswrap-linux.c
+@@ -8516,7 +8516,99 @@ PRE(sys_ioctl)
+       break;
+ 
+    case VKI_PERF_EVENT_IOC_ID:
+-      PRE_MEM_WRITE("ioctl(VKI_PERF_EVENT_IOC_ID)", (Addr)ARG3, sizeof(__vki_u64));
++      PRE_MEM_READ("ioctl(VKI_PERF_EVENT_IOC_ID)", (Addr)ARG3, sizeof(__vki_u64));
```

Comment:
This Perf change seems out of place in the v4v ioctl patch.

Reply:
Yes, out of place and incorrect.